tests/Resources/ti.ui.window.test.js

```diff
@@ -683,6 +678,85 @@ describe('Titanium.UI.Window', function () {
 			should(win.custom).eql(1234);
 		});
 
+		describe('#close()', () => {
+			it('is a Function', () => {
+				win = Ti.UI.createWindow({
+					backgroundColor: '#0000ff'
+				});
+
+				should(win).have.a.property('close').which.is.a.Function();
+			});
+
+			it('returns a Promise', finish => {
+				win = Ti.UI.createWindow({
+					backgroundColor: '#0000ff'
+				});
+
+				const openPromise = win.open();
+				openPromise.then(() => {
```

Comment:
<!--
  0 failure: 
  1 warning:  tests/Resources/t...
  
  
  DangerID: danger-id-Danger;
  File: tests/Resources/ti.ui.window.test.js;
  Line: 696;
-->

- :warning: tests/Resources/ti.ui.window.test.js line 696 – Each then() should return a value or throw (promise/always-return)